yb-voyager/cmd/export.go

```diff
@@ -97,7 +97,7 @@ func registerCommonExportFlags(cmd *cobra.Command) {
 		"provide SSL Root Certificate Revocation List (CRL)")
 
 	cmd.Flags().BoolVar(&startClean, "start-clean", false,
-		"clean the project's data directory for already existing files before start(Note: works only for export data command)")
+		"cleans up the project directory for schema or data files depending on the export command")
```

Comment:
clean the project's schema or data directory, depending on the export sub-command used

Reply:
done, as suggested by Amit